src/es/vcpvmp/src/eu/kanade/tachiyomi/extension/es/vcpvmp/VCPVMP.kt

```diff
@@ -27,25 +27,35 @@ open class VCPVMP(override val name: String, override val baseUrl: String) : Par
 
     override fun latestUpdatesNextPageSelector() = throw UnsupportedOperationException("Not used")
 
-    override fun popularMangaRequest(page: Int) = GET("$baseUrl/page/$page", headers)
+    override fun popularMangaRequest(page: Int) = GET("$baseUrl/$urlSuffix/page/$page", headers)
 
-    override fun popularMangaSelector() = "div#ccontent div.gallery"
+    override fun popularMangaSelector() = "div.blog-list-items > div.entry"
 
     override fun popularMangaFromElement(element: Element) = SManga.create().apply {
-        element.select("a.cover").first()!!.let {
+        element.select("a.popimg").first()!!.let {
             setUrlWithoutDomain(it.attr("href"))
-            title = it.select("div.caption").text()
+            title = it.select("img").attr("alt")
             thumbnail_url = it.select("img").attr("abs:src").substringBefore("?")
         }
     }
 
-    override fun popularMangaNextPageSelector() = "ul.pagination > li.active + li"
+    override fun popularMangaNextPageSelector() = "div.wp-pagenavi > span.current + a"
 
     override fun mangaDetailsParse(document: Document) = SManga.create().apply {
-        document.select("div#catag").let {
-            genre = document.select("div#tagsin > a[rel=tag]").joinToString { it.text() }
-            status = SManga.UNKNOWN
-            thumbnail_url = document.select(pageListSelector).firstOrNull()?.attr("abs:src")
+        document.select("div.tax_post").let {
+            status = SManga.COMPLETED
```

Comment:
Are these titles single-chapter always and never receive new updates? If so, consider adding `update_strategy` as well. In that way, these titles will be always skipped during global update as they will never receive any new chapters.

```suggestion
            status = SManga.COMPLETED
            update_strategy = UpdateStrategy.ONLY_FETCH_ONCE
```

You need to add the import as well

```kotlin
import eu.kanade.tachiyomi.source.model.UpdateStrategy
```